ivadomed/loader/adaptative.py

```diff
@@ -176,7 +177,7 @@ class BIDStoHDF5:
     """Converts a BIDS dataset to a HDF5 file.
 
     Args:
-        root_dir (str): Path to the BIDS dataset.
+        path_data (list) or (str): Path(s) to BIDS datasets
```

Comment:
using `list` as a variable name could be a naming conflict with the  built-in python `list()`. I would suggest maybe using `lst` instead?